src/python/grpcio/grpc/__init__.py

```diff
@@ -1832,6 +1832,30 @@ def local_server_credentials(local_connect_type=LocalConnectionType.LOCAL_TCP):
     return ServerCredentials(
         _cygrpc.server_credentials_local(local_connect_type.value))
 
+    
+def alts_channel_credentials(service_accounts=[]):
+    """Creates a ChannelCredentials for use with an ALTS-enabled Channel.
+
+    This is an EXPERIMENTAL API.
+
+    Args:
+      service_accounts: list of strings, target service accounts
```

Comment:
Could you please add the following description to `service_accounts`: 
"A list of server identities acceptable by the client. If target service accounts are provided and none of them matches the peer identity of the server, handshake will fail. The arg can be empty if the client does not have any information about trusted server identity."

Reply:
Added.